From 5d4a65f10a46726d4f69e01f27662dc485c85b32 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:40 -0400
Subject: [redhat] kabi: Adding symbol unlock_page (mm/filemap.c)

Message-id: <36f6f284a4c0c7fea2e10a7af2c2662900d9e6d4.1410510773.git.poros@redhat.com>
Patchwork-id: 96084
O-Subject: [PATCH RHEL7.1 220/233] kabi: Adding symbol unlock_page (mm/filemap.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Unlock a locked page.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/unlock_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/unlock_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/unlock_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/unlock_page b/redhat/kabi/kabi-module/kabi_ppc64/unlock_page
new file mode 100644
index 0000000..b76b7d6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/unlock_page
@@ -0,0 +1,2 @@
+#1-
+0x329c1aae unlock_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/unlock_page b/redhat/kabi/kabi-module/kabi_s390x/unlock_page
new file mode 100644
index 0000000..9672e24
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/unlock_page
@@ -0,0 +1,2 @@
+#1-
+0x4f88eeb8 unlock_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/unlock_page b/redhat/kabi/kabi-module/kabi_x86_64/unlock_page
new file mode 100644
index 0000000..81a37c6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/unlock_page
@@ -0,0 +1,2 @@
+#1-
+0x5de75127 unlock_page vmlinux EXPORT_SYMBOL
-- 
1.7.1